app/views/layouts/_navbar.html.erb

```diff
@@ -8,18 +7,14 @@
         <span class="icon-bar"></span>
         <span class="icon-bar"></span>
       </button>
-      <h4><%= link_to 'Logo', root_path %></h4>
+      <%= link_to image_tag("logo.jpg", width: '81', height: '57', :border=>0), root_path %>
```

Comment:
If you know the exact size you want your logo, why not resizing the original image to that size? also, `border:0` could be in the css better. As a rule of thumb try to never include styles directly in the html :)